tests/integration/Core/System/SalesChannel/Context/CartRestorerTest.php

```diff
@@ -540,13 +539,8 @@ private function createProduct(Context $context): string
         return $productId;
     }
 
-    private function createSalesChannelContext(string $contextToken, ?string $customerId = null): SalesChannelContext
+    private function createSalesChannelContext(string $contextToken): SalesChannelContext
     {
-        $salesChannelData = [];
-        if ($customerId) {
-            $salesChannelData[SalesChannelContextService::CUSTOMER_ID] = $customerId;
-        }
-
         return static::getContainer()->get(SalesChannelContextFactory::class)->create(
             $contextToken,
             TestDefaults::SALES_CHANNEL
```

Comment:
should `salesChannelData` not be passed here instead? :see_no_evil: strange, that it seems not to be important :grimacing: